articles/relnotes/index.md

```diff
@@ -17,6 +17,16 @@ For installation instructions, please refer to the [install guide](xref:microsof
 For update instructions, please refer to the [update guide](xref:microsoft.quantum.update).
 
 
+## Version 0.xx.xxxx.xxxx
+
+*Release date: March 26th, 2020*
+
+This release contains the following:
+
+- New support for access modifiers in Q#, for more information see [File Structures](xref:microsoft.quantum.language.file-structure)
```

Comment:
May be good to note that we switched to the LTS version of .NET Core, such that .NET Core SDK 3.1 is now required. (Please correct me if I'm wrong, @bettinaheim!)